app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java

```diff
@@ -1926,8 +1926,6 @@ private void searchFiles() {
           != PackageManager.PERMISSION_GRANTED) {
         if (ActivityCompat.shouldShowRequestPermissionRationale(this,
```

Comment:
I don't think there is any need to keep this empty `if` statement. Squash your commits into a single commit as there is no need of 2 commits for such a small change.

Reply:
:) Okay,I have removed it and squashed all three of my commits into one.